docs/governance/government/person-cred-doc.md

```diff
@@ -0,0 +1,424 @@
+# Person – Credential Documentation
+
+## About this Credential
+
+This document describes the Government of British Columbia’s Person credential to help potential verifiers determine whether it is suitable for their needs. The intended audience includes policy analysts, privacy specialists, solution architects, developers, and data architects.
+
+### Version History
+
+| Ver.       | Date        | Notes                                               |
+| ---------- | ----------- | --------------------------------------------------- |
+| <b>1.0</b> | 25-Oct-2023 | -                                                   |
+| <b>1.1</b> | 28-Mar-2023 | Corrected Schema ID, updated description of Holders |
+
+## Credential Overview
+
+The Person credential is a verifiable credential (VC) issued to individuals to enable them to prove their identity and to digitally provide key personal information – name, data of birth, photo, address – to other parties (“verifiers”). The Person credential is intended to be used as a source of trusted identity information in a wide range of contexts, either alone or in conjunction with other credentials. The Person credential currently is only issued to the BC Wallet, a mobile application published by the Digital Identity & Trust Program (DITP) within the Office of the Chief Information Officer (OCIO) of the BC
```

Comment:
Be consistent in defining the Person credential as a collection of personal information such as their "name**S***" with the "names" plural.